optaplanner-persistence/optaplanner-persistence-jaxb/pom.xml

```diff
@@ -63,8 +63,17 @@
       <artifactId>jaxb-impl</artifactId>
     </dependency>
     <dependency>
-      <groupId>javax.activation</groupId>
-      <artifactId>activation</artifactId>
+      <groupId>jakarta.activation</groupId>
+      <artifactId>jakarta.activation-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish</groupId>
+      <artifactId>jakarta.json</artifactId>
```

Comment:
Can we make these new dependencies test dependencies?
In fact, some of the old dependencies should probably be test dependencies.
We want to provide XmlAdaptors for score, implemented against the JAXB API.
We don't want to choose a JAXB implementation for the users, only for our testing.

Reply:
And we certainly don't want to bloat the classpath.
Having both com.sun.xml.bind and org.glassfish and org.eclipse.persistence dependencies, is suspicious.
A lot of com.sun things became org.glassfish things at some point. Nowadays everything should be in jakarta.
To settle these questions, look at the master of the resteasy project. Or the quarkus-resteasy-jaxb project. What is in their classpath?